base_layer/wallet/src/utxo_scanner_service/utxo_scanner_task.rs

```diff
@@ -66,6 +66,7 @@ use crate::{
     },
 };
 
+pub const BIRTHDAY_GENESIS_FROM_UNIX_EPOCH: u64 = 1640995200; // seconds since 2022-01-01 00:00:00 UTC
```

Comment:
This shouldn't be a hard-coded value. It needs to match the [value](https://github.com/tari-project/tari/blob/43efc07a9713ebb498ca635603b97c34b561b1e2/base_layer/key_manager/src/cipher_seed.rs#L54) corresponding to the `CipherSeed` version in question (of which there is currently only one).

Reply:
I was actually playing around to test it. The best approach, imo, is to add a method to get the birthday date from unix epoch, like `get_birthday_time_from_unix()`.